src/main/java/seedu/moneygowhere/common/Messages.java

```diff
@@ -50,5 +50,14 @@ public class Messages {
             + "File loaded successfully :)";
     public static final String LOCAL_STORAGE_ERROR_SAVE_DATA = ""
             + "There is an error in saving file...\n"
-            + "File does not exist or Path is wrong :(";
+            + "File does not exist or Path is wrong :("
+            + "SYNTAX: View-Expense [-i EXPENSE_NUMBER]";
```

Comment:
I believe that this string isn't part of the LOCAL_STORAGE_ERROR_SAVE_DATA variable. Could you remove it please?

Reply:
Oh good catch, thank you for pointing it out!